com.unity.render-pipelines.core/CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fixed normal bias field of reference volume being wrong until the profile UI was displayed.
 - Fixed L2 for Probe Volumes.
 - Fixed assertion on compression of L1 coefficients for Probe Volume.
+- Fixed missing increment/decrement controls from DebugUIIntField & DebugUIUIntField widget prefabs.
```

Comment:
Was this issue something that is reproduced in previous versions or only in current version?

Reply:
This was broken in the branch I started on and I assumed it was broken, but you're right - I should verify this was broken in a shipped version as well, and only include this in release notes if that's the case.

Reply:
Checked and this was missing from HDRP in 2020.2.3, so I'd say let's keep it. It wasn't a problem before because a) there's only a few int/uint fields to begin with, and b) there was no mouse support and you could still increment/decrement with keyboard left/right.